src/ErrorsReadInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Validator;
+
+interface ErrorsReadInterface
```

Comment:
Suggest remove this interfaceand in future prs merge `ResultSet` and `Result` to one class.

Reply:
:heavy_check_mark:

Reply:
We could remove it in the same future PR too, but OK, removed.